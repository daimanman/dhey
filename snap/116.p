{"C":1,"Duration":114691376,"EndTime":1551409839947274800,"Err":"","FileId":"","InputFileName":"","LonhApi":"findFiregroundpm","Method":"","N":1,"P":null,"Payload":"","QH":{},"Remark":"","ReqNums":1,"StartTime":1551409839832583400,"Status":1,"TaskId":116,"Type":"LONH","Url":"","Z":""}