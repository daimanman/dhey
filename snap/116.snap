{"AvgTotal":0.1057155,"Fastest":0.1057155,"Slowest":0.1057155,"Average":0.1057155,"Rps":9.37097340986295,"AvgConn":0.0488755,"AvgDNS":0.0009967,"AvgReq":0.0010116,"AvgRes":0,"AvgDelay":0.0558284,"ConnMax":0.0009967,"ConnMin":0.0009967,"DnsMax":0.0009967,"DnsMin":0.0009967,"ReqMin":0.0010116,"ReqMax":0.0010116,"ResMin":0.0010116,"ResMax":0,"DelayMax":0.0558284,"DelayMin":0.0558284,"Lats":[0.1057155],"ConnLats":[0.0009967],"DnsLats":[0.0009967],"ReqLats":[0.0010116],"ResLats":[0],"DelayLats":[0.0558284],"Total":106712500,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":0,"SizeReq":0,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.1057155,"Count":1,"Frequentcy":1},{"Mark":0.1057155,"Count":0,"Frequentcy":0},{"Mark":0.1057155,"Count":0,"Frequentcy":0},{"Mark":0.1057155,"Count":0,"Frequentcy":0},{"Mark":0.1057155,"Count":0,"Frequentcy":0},{"Mark":0.1057155,"Count":0,"Frequentcy":0},{"Mark":0.1057155,"Count":0,"Frequentcy":0},{"Mark":0.1057155,"Count":0,"Frequentcy":0},{"Mark":0.1057155,"Count":0,"Frequentcy":0},{"Mark":0.1057155,"Count":0,"Frequentcy":0},{"Mark":0.1057155,"Count":0,"Frequentcy":0}],"TaskId":116,"Duration":0.1067125}